core/common/lib/policy-engine-lib/src/test/java/org/eclipse/edc/policy/engine/PolicyEngineImplTest.java

```diff
@@ -213,6 +215,25 @@ void validateRuleFunctionOutOfScope() {
         assertThat(policyEngine.evaluate("bar", policy, context).succeeded()).isTrue();
     }
 
+    @ParameterizedTest
+    @ValueSource(booleans = { true, false })
+    void validateAllScopesPrePostFunctionalValidator(boolean preValidation) {
```

Comment:
nit: influencing the test code flow with a parameter of a parameterized test seems spurious, I would split this in two tests